public/index.html

```diff
@@ -2,42 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
%PUBLIC_URL% - its the right thing. Not necessary to delete.

Reply:
the path is different